### PR TITLE
Upgrade to play 2.6

### PR DESCRIPTION
**Pull request by DamienButterworth**

### CODE REVIEW

**Review by retroredge — APPROVED** [review]
LGTM

File: build.sbt [hunk 1]
```diff
@@ -20,17 +21,27 @@ lazy val microservice = Project(appName, file("."))
 
 val compileDeps = Seq(
 
-  "uk.gov.hmrc"             %% "simple-reactivemongo"     % "7.20.0-play-25",
+  "uk.gov.hmrc"             %% "simple-reactivemongo"     % "7.30.0-play-26",
   "org.sangria-graphql"     %% "sangria"                  % "1.4.2",
-  "org.sangria-graphql"     %% "sangria-play-json"        % "1.0.0",
-  "uk.gov.hmrc"             %% "bootstrap-play-25"        % "4.16.0"
+  "org.sangria-graphql"     %% "sangria-play-json"        % "1.0.3",
+  "uk.gov.hmrc"             %% "bootstrap-play-26"        % "1.11.0"
```
Comment (chrisjameswright) [inline build.sbt:27]: Was there a reason for picking this specific version?

File: test/uk/gov/hmrc/customs/datastore/services/EoriStoreSpec.scala [hunk 2]
```diff
@@ -17,18 +17,17 @@
...
-import uk.gov.hmrc.customs.datastore.config.AppConfig
 import uk.gov.hmrc.customs.datastore.domain._
+import uk.gov.hmrc.customs.datastore.utils.SpecBase
 import uk.gov.hmrc.mongo.MongoConnector
 
 import scala.concurrent.ExecutionContext.Implicits.global
 import scala.concurrent.Future
 
-class EoriStoreSpec extends WordSpec with MustMatchers with MongoSpecSupport with DefaultAwaitTimeout with FutureAwaits with BeforeAndAfterEach {
+class EoriStoreSpec extends SpecBase {
```
Comment (derekspalmer) [inline test/uk/gov/hmrc/customs/datastore/services/EoriStoreSpec.scala:30]: :+1:

**Review by derekspalmer — APPROVED** [review]
LGTM 
We'll need to bump up sbt, Scala versions & build jobs.

**Comment by chrisjameswright** [comment]
Is there an equivalent to the tests that have been removed, or are we intentionally lose these? (e.g., `test/uk/gov/hmrc/customs/datastore/controllers/GraphQLControllerIt.scala`)

File: build.sbt (hunk 1 above)
Comment (chrisjameswright) [reply build.sbt:27]: Also, should the move be to [bootstrap-play](https://github.com/hmrc/bootstrap-play) instead? It isn't necessary, but looks like it shouldn't be particularly more effort.

**Review by chrisjameswright — APPROVED** [review]

**Pull request merged**